# 0xF1 : Status Message - Serial Number

# Variables declaration
	uint8 SyncByte  # IMU Address
	uint8 MessageID  # Message ID
	uint8[8] serial_number  # Serial Number
	uint16 Checksum  # Uint16 Checksum. Complement = False
